extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -514,6 +520,16 @@ export class InstrumentationService {
     this.win_.setTimeout(this.win_.clearInterval.bind(this.win_, intervalId),
         maxTimerLength * 1000);
   }
+
+  isTriggerAllowed_(triggerType, element) {
+    if (!isInIframe(element) ||
```

Comment:
I'd like to focus more on "embed" and less on "iframe" in terminology here. Or at the very least "iframe embed". The right implementation will be altogether different and simply:
```
isIframeEmbed = element.ownerDocument.defaultView != analyticsService.win;
```

I don't see a huge need in moving this check into `iframe-helper.js`. But if you'd like a better place would be either `service.js` near `getTopWindow` or `friendly-iframe-embed.js`